src/pages/Notification/NotificationPage.jsx

```diff
@@ -0,0 +1,119 @@
+import styled from 'styled-components';
+import { NavLink } from 'react-router-dom';
+import React, { useState } from 'react';
+import NotificationItem from './NotificationItem';
+
+const Notification = () => {
+  const [notifications, setNotifications] = useState([
+    {
+      name: '닉네임',
+      message: '님과 친구가 되었습니다',
+      time: '00분 전',
+    },
+    {
+      name: '펀딩 이름',
+      message: '펀딩이 종료되었습니다',
+      time: '00시간 전',
+    },
+  ]);
+  return (
+    <BodyContainer>
+      <HeaderWord>알림</HeaderWord>
+      <NavContainer>
+        <HeadBtn>
+          <StyledNavLink exact to='/' activeClassName='active'>
+            전체
+          </StyledNavLink>
+        </HeadBtn>
+        <HeadBtn>
+          <StyledNavLink to='/friend' activeClassName='active'>
+            친구
+          </StyledNavLink>
+        </HeadBtn>
+        <HeadBtn>
+          <StyledNavLink to='/funding' activeClassName='active'>
+            펀딩
+          </StyledNavLink>
+        </HeadBtn>
+      </NavContainer>
+      <ContentContainer>
+        <EachContainer>
+          {notifications.map((notification) => (
+            <NotificationItem
+              name={notification.name}
+              message={notification.message}
+              time={notification.time}
+            ></NotificationItem>
+          ))}
+        </EachContainer>
+      </ContentContainer>
+    </BodyContainer>
+  );
+};
+
+export default Notification;
+
+const BodyContainer = styled.div`
+  width: 375px;
+  height: 812px;
+  top: 160px;
+  left: 160px;
+`;
+
+const HeaderWord = styled.div`

```

Comment:
시맨틱 태그 활용하면 더욱 좋겠습니다~
```suggestion
const HeaderWord = styled.header`
```